src/data/faq_de.json

```
@@ -128,16 +128,16 @@
                 "id": "eu_dcc",
                 "accordion": [
                     {
-                    "title": "Welche EU-Zertifikate werden unterstützt?",
-                    "anchor": "eu_dcc_which",
-                    "active": true,
-                    "textblock": [
-                        "Ab <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> der Corona-Warn-App (CWA) werden alle drei Arten der EU digitalen COVID-Zertifikate unterstützt:",
-                        "<ul>  <li>digitales Impfzertifikat</li>  <li>digitales Genesenenzertifikat</li>  <li>digitales Testzertifikat</li> </ul>",
-                        "Bisher wurden digitale Impfzertifikate erstmalig mit <a href='../../de/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>Version 2.3</a> unterstützt, gefolgt von der zusätzlichen Unterstützung von digitalen Testzertifikaten in <a href='../../de/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>Version 2.4.</a> Mit der Veröffentlichung der <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> wurden digitale Genesenenzertifikate zusätzlich unterstützt.",
-                        "Die Verordnung über das <a href='https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_de' target='_blank' rel='noopener noreferrer'>digitale COVID-Zertifikat der EU</a> ist am 1. Juli 2021 in Kraft getreten. Das digitale COVID-Zertifikat der EU wird in allen EU-Mitgliedstaaten anerkannt."
-                    ]
-                  },
+                        "title": "Welche EU-Zertifikate werden unterstützt?",
+                        "anchor": "eu_dcc_which",
+                        "active": true,
+                        "textblock": [
+                            "Ab <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> der Corona-Warn-App (CWA) werden alle drei Arten der EU digitalen COVID-Zertifikate unterstützt:",
+                            "<ul><li>digitales Impfzertifikat</li><li>digitales Genesenenzertifikat</li><li>digitales Testzertifikat</li></ul>",
+                            "Bisher wurden digitale Impfzertifikate erstmalig mit <a href='../../de/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>Version 2.3</a> unterstützt, gefolgt von der zusätzlichen Unterstützung von digitalen Testzertifikate in <a href='../../de/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>Version 2.4.</a> Mit der Veröffentlichung der <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> wurden digitale Genesenenzertifikate zusätzlich unterstützt.",

```

Why is the correction here being overwritten? 
Testzertifikaten

---

```suggestion
                            "Bisher wurden digitale Impfzertifikate erstmalig mit <a href='../../de/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>Version 2.3</a> unterstützt, gefolgt von der zusätzlichen Unterstützung von digitalen Testzertifikaten in <a href='../../de/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>Version 2.4.</a> Mit der Veröffentlichung der <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> wurden digitale Genesenenzertifikate zusätzlich unterstützt.",
```
Re-applied grammar correction.